src/ncp/ncp_openthread.hpp

```diff
@@ -164,6 +164,8 @@ class ControllerOpenThread : public MainloopProcessor
     ~ControllerOpenThread(void) override;
 
 private:
+    using Clock = std::chrono::steady_clock;
```

Comment:
Please reuse the definition in `src/common/time.hpp`, reusing the same pre-definition makes sure that we don't use `std::chrono::system_clock` for a Clock usage.

Reply:
Done